main.css

```diff
@@ -315,3 +392,396 @@ li {
 }
+
+.informationsection {
+  background-color: white;
+  color: #172b4d;
+  padding: 120px 20px 8px 20px;
+}
+
+.informationtitle {
+  display: flex;
+  justify-content: center;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  margin: 0 0 30px 0;
+}
+
+.informationbutton {
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px auto;
+  width: 131px;
+  height: 48px;
+}
+
+.informationbutton:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  box-sizing: border-box;
+}
+
+.informationbutton:active {
+  background: #e05304;
+}
+
+.informationbutton:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+.informationsection input {
+  margin: 0 25px 0 25px;
+  width: 90%;
+  border: none;
+}
+
+.underline2 {
+  border: 1px solid #212121;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  height: 1px;
+  width: 90%;
+  display: block;
+  margin: 0 auto;
+}
+
+.undeline2line {
+  margin: 0 20px 0 20px;
+}
+
+.textareainformation {
+  margin: 30px 20px 10px 20px;
+  width: 90%;
+  border: none;
+  background-color: #fbf8f7;
+  border-radius: 8px;
+}
+
+.socialmedia2 {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.black-line {
+  display: block;
+  width: 134px;
+  height: 5px;
+  margin: 23px auto;
+}
+
+.gray-line,
+.aboutmegray-line {
+  width: 100%;
+  height: 1px;
+}
+
+.storybotton {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  margin: 20px 20px 0 20px;
+  background-color: #ff6b00;
+  width: 124px;
+  height: 48px;
+  color: #fff;
+}
+
+.storybotton:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  box-sizing: border-box;
+}
+
+.storybotton:active {
+  background: #e05304;
+}
+
+.staybotton:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+@media screen and (min-width: 992px) {
+  .ulnavbar {
+    display: none;
+  }
+
+  .firstsection {
+    display: flex;
+    flex-direction: row;
+    background-image: url(./images/Desktop/first-page/Illustration-header.svg);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: 300px;
+    padding: 120px 40px 40px 40px;
+  }
+
+  .firstsection-column2 {
+    order: -1;
+    width: 10%;
+    height: 100%;
+    align-self: center;
+  }
+
+  .firstsection-column1 {
+    width: 55%;
+    align-self: center;
+    padding-top: 70px;
+  }
+
+  .socialmedia {
+    flex-direction: column;
+    padding: 10px 0 0 0;
+    margin: 10px 0 10px 0;
+  }
```

Comment:
- [ ] kindy add gaps between social media icons to improve your UI design.
<img width="107" alt="image" src="https://user-images.githubusercontent.com/77190203/154492136-877e0f25-34ed-4227-a65b-142a4dead604.png">